test/source/mock/google/strategies/send-message-strategy.ts

```diff
@@ -7,15 +7,23 @@ import { GoogleData } from '../google-data';
 import { HttpClientErr } from '../../lib/api';
 import { PgpMsg } from '../../../core/pgp-msg';
 
+// TODO: Make a better structure of ITestMsgStrategy. Because this class doesn't test anything, it only saves message in the Mock
+class SaveMessageInStorageStrategy implements ITestMsgStrategy {
```

Comment:
I mildly remember a similar method before, will have a look

Reply:
I think I refactored it away that time

Reply:
We should not define this as `SaveMessageInStorageStrategy`. Instead, all sent messages should be saved in storage.

Also, each test should have its own storage so that messages sent from one test don't interfere with messages in other tests. Right now all tests share storage. Also, each storage should be wiped between test retries.